frontend/workflows/ec2/src/resize-asg.tsx

```diff
@@ -87,26 +87,24 @@ const GroupDetails: React.FC<WizardChild> = () => {
 const Confirm: React.FC<ConfirmChild> = ({ notes }) => {
   const group = useDataLayout("groupData").displayValue();
   const resizeData = useDataLayout("resizeData");
+  const formattedNotes = notes?.map(note => <div>{note.text}</div>);
 
   return (
     <WizardStep error={resizeData.error} isLoading={resizeData.isLoading}>
-      <Confirmation action="Resize" />
-      <Paper style={{ width: "100%", margin: "2.5% 0" }} elevation={3}>
-        <MetadataTable
-          data={[
-            { name: "Name", value: group.name },
-            { name: "New Min Size", value: group.size.min },
-            { name: "New Max Size", value: group.size.max },
-            { name: "New Desired Size", value: group.size.desired },
-          ]}
-        />
-      </Paper>
-      <NotePanel notes={notes} />
+      <Confirmation action="Resize">{notes && formattedNotes}</Confirmation>
+      <MetadataTable
+        data={[
+          { name: "Name", value: group.name },
+          { name: "New Min Size", value: group.size.min },
+          { name: "New Max Size", value: group.size.max },
+          { name: "New Desired Size", value: group.size.desired },
+        ]}
+      />
```

Comment:
didn't add the accordion since we don't have metadata like annotations/labels for an [ASG](https://github.com/lyft/clutch/blob/main/api/aws/ec2/v1/ec2.proto#L67)